content/ember-1-0-prerelease.html.md

```diff
@@ -1,14 +1,12 @@
 ---
-title: ''
-authors: []
-date: 2012-08-03T00:00:00.000Z
-tags: []
----
---- 
 title: Ember 1.0 Prerelease
-author: Peter Wagenet
-tags: Releases, Version 1.x, Prerelease, 2012, Router
```

Comment:
I removed `Prerelease` and `Router` based on a similar file `content/ember-1-0-pre2.html.md`.